static/js/shortcuts.js

```diff
@@ -1,5 +1,10 @@
 // Press arrow keys to turn a page
 (() => {
+  // Make code blocks focusable, so we can use activeElement to check if user is scrolling inside a code block
+  document.querySelectorAll("pre").forEach((element) => {
+    element.setAttribute("tabindex", "0");
```

Comment:
`tabIndex` 的 `I` 应该大写：

```suggestion
    element.tabIndex = 0;
```